src/sentry/static/sentry/app/components/events/interfaces/spans/spanTree.tsx

```diff
@@ -335,4 +340,18 @@ const TraceViewContainer = styled('div')`
   border-bottom-right-radius: 3px;
 `;
 
+function isJavaScriptSDK(SDKName: string | undefined): boolean {
+  if (!SDKName) {
+    return false;
+  }
+
+  SDKName = SDKName.toLowerCase();
+
+  // based on
+  // https://github.com/getsentry/sentry-javascript/blob/master/packages/browser/src/version.ts
+  // https://github.com/getsentry/sentry-javascript/blob/master/packages/node/src/version.ts
+
+  return SDKName === 'sentry.javascript.browser' || SDKName === 'sentry.javascript.node';
```

Comment:
Should we be targetting node server apps?

Reply:
`sentry.javascript.node` is node server app

Reply:
Confirmed with @HazAT to show missing instrumentation for node apps.  https://github.com/getsentry/sentry/pull/17887/commits/2dde48eb252bfcdf9f587c349aa784fb7328d66b